app/helpers/application_helper/toolbar/miq_ae_tools_simulate_center.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationHelper::Toolbar::MiqAeToolsSimulateCenter < ApplicationHelper::
       N_('Copy object details for use in a Button'),
       N_('Copy'),
       :url       => "resolve",
-      :url_parms => "?button=copy"),
+      :url_parms => "?button=copy",
+      :klass     => ApplicationHelper::Button::ButtonWithoutRbacCheck),
```

Comment:
No, I am not sure what that was meant for.  Maybe @dclarizio remembers why it is there.